rmw_fastrtps_shared_cpp/src/rmw_node.cpp

```diff
@@ -327,6 +346,27 @@ __rmw_create_node(
         Property(
           "dds.sec.access.builtin.Access-Permissions.permissions", security_files_paths[5]));
 
+      std::string security_logging_file_path;
+      if (get_security_file_path(security_logging_file_path,
+                                 "logging.xml", security_options->security_root_path)) {
+
+        std::string distribute, event_log_level, log_file;
+        if (load_security_logging_options(distribute, event_log_level, log_file, security_logging_file_path)) {
+          property_policy.properties().emplace_back(
+            Property("dds.sec.log.plugin", "builtin.DDS_LogTopic"));
+          property_policy.properties().emplace_back(
+            Property("dds.sec.log.builtin.DDS_LogTopic.distribute", distribute));
+          property_policy.properties().emplace_back(
+            Property("dds.sec.log.builtin.DDS_LogTopic.event_log_level", event_log_level));
```

Comment:
This *is* `log_level` as defined in section 8.6.2.1.1. We have temporarily renamed it `event_log_level` to avoid confusion with `LoggingLevel` defined in section 9.6 until we figure out if they actually are the same thing and the spec has a mistake.